tests/unit/backend/test_headers/header2.hxx

```diff
@@ -0,0 +1,12 @@
+#ifndef HEADER_2
+#define HEADER_2
+
```

Comment:
#include <string>

Reply:
ROOT interpreter already includes `string` library right @etejedor ?

Reply:
That doesn't throw an error too

Reply:
Yes, it includes it and it also makes the std namespace available. But I prefer to write complete headers that also compile without the ROOT interpreter :)

Reply:
Ok @etejedor . Will make this change 👍